README.rst

```diff
@@ -124,18 +124,33 @@ sites)::
   ./manage.py lms collectstatic
   ./manage.py cms collectstatic
 
+Setup Studio SSO for Development::
```

Comment:
```suggestion
Set up CMS SSO for Development::
```

two more nits-- set up as a verb is two words, and CMS is the name of the django service that includes Studio, despite the fact that we call the whole service Studio about 25% of the time